frontend/src/i18n/locales/de.yaml

```diff
@@ -6,3 +6,22 @@ navigation: Navigation
 page-not-found:
   title: Seite nicht gefunden
   body: Die angefragte Resource existiert nicht. Sorry! :-(
+
+about:
+  title: Information
+  body: >
+    Tobira ist ein Videoportal für Opencast. Es erlaubt seinen Nutzern, alle
+    Episoden und Serien aus der verbundenen Opencast-Instanz zu durchstöbern,
+    anzuschauen und zu organisieren. Tobira, wie auch Opencast selber, ist
+    kostenlose Open-Source-Software. Weitere Informationen und die Möglichkeit,
+    Fehler zu melden, gibt es im <1>GitHub Repository des Projekts</1>.
+
+main-menu:
+  label: Menü
+  theme: Design
```

Comment:
Much German. Also is this really the same? :thinking: 

We don't know yet what will really hind behind this button, I guess, but maybe "Farbschema"?

Reply:
Since we don't know what will be behind it, I would suggest also just keeping it for now. We can still change it later once we know more. Also, I got this "translation" from YouTube where "theme" is translated to "Design".

Reply:
Fair enough.